index.html

```diff
@@ -2,12 +2,65 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    <div class="wrapper">
 
+        <header class="toolbar">
+            <nav>            
+                <a href="#">
+                <div class="item">Edahí</div> </a>        
+                <i class="fa-solid fa-bars"></i>                    
+            </nav>
+        </header>
+        
+        <section>
+            <p class="info">Hey There. <br>
+                I’m Edahí</p>
```

Comment:
- [Optional] Please change the to a `<h1>`. This is HTML semantics best practice.